Makefile

```diff
@@ -3,16 +3,26 @@ export GOPATH=/tmp/go
 endif
 
 INSTALL := /usr/bin/install
+DBDIR := /var/run/redis/sonic-db/
 
 all: sonic-telemetry
 
 sonic-telemetry:
+# copy sonic-telemetry source code into ${GOPATH}/src directory for building, otherwise it is not using committed codes
```

Comment:
>otherwise it is not using committed codes [](start = 78, length = 41)

what is it using?

Reply:
Go will use ${GOPATH}/src as the building dir, we need to copy the modified codes there. Before the changes,  it is download the source codes from the latest at github.com/Azure/sonic-telemetry/XXXX instead of using the local PR committed codes.